fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/service/ChargeWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -287,7 +287,7 @@ private boolean isAnyLoanProductsAssociateWithThisCharge(final Long chargeId) {
     }
 
     private boolean isAnySavingsProductsAssociateWithThisCharge(final Long chargeId) {
-        final String sql = "select (CASE WHEN (exists (select 1 from m_savings_product_charge sc where sc.charge_id = ?)) = 1 THEN 'true' ELSE 'false' END)";
+        final String sql = "select (CASE WHEN exists (select 1 from m_savings_product_charge sc where sc.charge_id = ?) THEN 'true' ELSE 'false' END)";
```

Comment:
This is also unrelated.

Reply:
This is a bug but anyway It was restored